game/definitions.rpy

```diff
@@ -1023,21 +1016,13 @@ init -990 python in jn_globals:
         "bollock",
         "bollok",
         "boner",
-        "boob",
-        "boobs",
-        "booobs",
-        "boooobs",
-        "booooobs",
-        "booooooobs",
+        "boob(?!er|on)",
```

Comment:
Maybe this could be `b[0o]+b(?!er|on)`?

Seems like it'd aldo catch the `b00bs` above along with combinations of o/0 at any length